muDIC/mesh/meshUtilities.py

```diff
@@ -399,11 +399,18 @@ def gen_node_positions(self):
             logger.exception("Mesh generation failed")
 
     def visualize_node(self):
-        import matplotlib.pyplot as plt
-        plt.plot(self.xnodes,self.ynodes,"o")
+        from PIL import ImageFont, Image, ImageDraw
+
+        im = Image.open(r'./example_data/ut_151.tif')
+
+        font = ImageFont.truetype("/Library/Fonts/Times New Roman.ttf", 40)
```

Comment:
This Font ("police") is not available in all OSes. For example, on Linux, it does not work. Instead of using a specific font, we can use the default font of the OS on the computer:
`font = ImageFont.load_default()`
Please update this line.